src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/PermalinkService.cs

```diff
@@ -97,14 +109,38 @@ private static JsonSerializerSettings BuildJsonSerializerSettings()
         private async Task<PermalinkViewModel> BuildViewModel(Permalink permalink)
         {
             var subject = await _subjectRepository.Get(permalink.Query.SubjectId);
-            var isValid = subject != null && await _subjectRepository.IsSubjectForLatestPublishedRelease(subject.Id);
+
+            var isValid = subject != null
+                          && !IsSuperseded(subject.Id)
+                          && await _subjectRepository.IsSubjectForLatestPublishedRelease(subject.Id);
 
             var viewModel = _mapper.Map<PermalinkViewModel>(permalink);
 
             viewModel.Invalidated = !isValid;
 
             return viewModel;
         }
+
+        private bool IsSuperseded(Guid subjectId)
+        {
+            var releaseId = _statisticsDbContext.ReleaseSubject
+                .Where(rs => rs.SubjectId == subjectId)
+                .Select(rs => rs.ReleaseId)
+                .Single();
+
+            var publication = _contentDbContext.Releases
+                .Include(r => r.Publication)
+                .Where(r => r.Id == releaseId)
+                .Select(r => r.Publication)
+                .Single();
+
+            return publication.SupersededById != null
+                   && _contentDbContext.Releases
+                       .Include(p => p.Publication)
+                       .Where(r => r.PublicationId == publication.SupersededById)
+                       .ToList()
+                       .Any(r => r.IsLatestPublishedVersionOfRelease());
```

Comment:
Has the publication got _all_ of it's Releases included to be able to do this check? I suspect it might only have one of it's releases loaded.